README.rst

```diff
@@ -0,0 +1,183 @@
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognise that the move to Collections is a big change in how Ansible is developed and delivered. Early cuts, which maybe will (read will) be broken from time to time.
+
+See the `Ansible Communication channels <https://docs.ansible.com/ansible/latest/community/communication.html>`_ for a list of IRC channels and email lists you can use to join the discussion. For specific issues please `raise a bug report <https://github.com/ansible/ansible/issues/new/choose>`_.
+
+Where we've come from, and where we are going
+=============================================
+
+To understand where we are going, it's often useful to remind ourselves of where are today.
+
+Ansible 2.9 and earlier
+------------------------
+
+**Classic Ansible**
+
+* Single repository `ansible/ansible <https://github.com/ansible/ansible>`_.
+* Single packages called `ansible`
+* `ansible` has major releases twice a year
+* New features go into the next major release, ie you may have to wait 6 months to take advantage.
+
+
+Ansible 2.10 and later
+----------------------
+
+* The ``ansible/ansible`` repository will only contain:
+
+  * The core Ansible programs, ansible-(playbook,galaxy,docs,test,etc)
+  * Documentation
+  * a tiny subset of supported modules and plugins
+  * Together this will be known as ``ansible-base``
+* The rest of the modules and plugins will move into multiple Collections
+* Collections can:
+
+  * Be release independently of Ansible, ie no more waiting 6 months for your new feature
+  * Have their own repo (GitHub, GitLab, etc) with dedicated backlog, ie no more shared massive issue & PR backlog
+  * Should still have CI testing
+  
+* Ansible 2.10 will pull in ``ansible-base`` and the various community collections.
+
+
+Terminology
+===========
+
+
+Classic Ansible (CA)
+  What we think of as Ansible prior to 2020. The open source project and the deliverable. What you currently get (as of Ansible 2.9) when you type “pip install ansible”. Currently developed almost entirely within the ``ansible/ansible`` repository on GitHub. 
+  
+Collection
+  A packaging format for bundling and distributing Ansible content: plugins, roles, modules. Can be released independently to other collections or Ansible so features can be made available sooner to users. Installed via ``ansible-galaxy collection install <namespace.collection>``.
+
+Ansible Base
+  The codebase that will be contained in github.com/ansible/ansible for the Ansible 2.10 release. Contains only the minimal amount of modules and plugins, allows other collections to be installed. Similar to Ansible 2.9 though without any content that has since moved into a collection. Only contains a small set of modules and plugins. A work in progress can be found in `ansible-base <https://github.com/ansible-collection-migration/ansible-base/>`_ repository.
```

Comment:
s/Contains only the minimal amount/Contains a minimal amount of modules and plugins

Also can probably drop the 'Only contains a small set of modules and plugins.' line.

Reply:
Will there be a pip package to install just `ansible-base`? E.g. `pip install ansible-base`? I can see many situations where it would be useful to get just base and not 'Community Content Collections'.